net.lutris.Lutris.yml

```diff
@@ -94,6 +94,8 @@ modules:
         path: patches/lutris-flatpak-support-pr2106.patch
       - type: patch
         path: patches/lutris-appdata-tags-pr2134.patch
+      - type: patch
+        path: patches/game.py.patch
```

Comment:
The patch name should be self-descriptive, telling what it does, e.g. "remove path to libgamemodeauto". And prefix it's name with `lutris-`.

Reply:
Renamed to `lutris-libgamemodeauto-path.patch`